src/solver/conv_asm_implicit_gemm_gtc_wrw_nhwc.cpp

```diff
@@ -864,6 +864,25 @@ bool ConvAsmImplicitGemmGTCDynamicWrwXdlopsNHWC::IsApplicable(
         return false;
 #endif
 
+#if WORKAROUND_ISSUE_2867
+    {
```

Comment:
[Notice] @bghimireamd Adding this would allow re-using the existing `MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_WRW_GTC_XDLOPS_NHWC` env var to override the W/A.
```C++
if(!miopen::IsEnabled(ENV(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_WRW_GTC_XDLOPS_NHWC)))
{
    ... // checks
       return false;
}
```
This is useful for checking the library with newer versions of HIP runtime, compiler etc without need to rebuild & reinstall MIOpen. I am not sure if this is necessary for this specific case, just to let you know.